BootloaderCommonPkg/Library/IppCryptoLib/rsa_verify.c

```diff
@@ -17,7 +17,8 @@
 /* Wrapper function for RSAVerify to make the inferface consistent.
  * Returns non-zero on failure, 0 on success.
  */
-int VerifyRsaSignature (const void *hash, const void *rsa_n,  const void *rsa_e, const void *sig)
+int VerifyRsaSignature (const void *hash, const void *rsa_n,  const void *rsa_e, const void *sig,

```

Comment:
Can we pass in  PUB_KEY_HDR and SIGNATUR_HDR instead ?
It is our implementation code,  should be fine to use SBL structure.

Reply:
I would remove sub function VerifyRsaSignature(). Will maintain all the functionality in RsaVerify_Pkcs_1_5() API.

Reply:
OK, that will be good.

Reply:
We maintain IPP macro's for return values in VerifyRsaSignature(). It still good to maintain wrapper function RsaVerify_Pkcs_1_5. I would update to pass UB_KEY_HDR and SIGNATUR_HDR to next level.